Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM node:5.1.1
+FROM ubuntu
```

Comment:
the red is what's removed akhi - i set it back to be `FROM ubuntu` as that's what we're running in production right now, and i manually install node accordingly.